package.json

```diff
@@ -56,7 +56,11 @@
     "npm": ">=7.21.0"
   },
   "dependencies": {
-    "lit": "^2.0.0"
+    "github-markdown-css": "^5.1.0",
```

Comment:
Please put these dependencies in devDependencies and make them absolute (remove the `^`)